### PR TITLE
feat: Add Ai Model Tracker To Threads

### PR DESCRIPTION
**Pull request by TopETH**
## Summary by CodeRabbit

- **New Features**
  - Introduced a new `models` table in the `public` schema with `model_name` and `model_value` columns.
  - Added a new `model` column to the `public.thread` table with default values and foreign key constraints.
- **Permissions**
  - Defined select permissions for roles `anonymous` and `user` on the `model_name` and `model_value` columns in the `models` table.
- **Data Initialization**
  - Inserted initial rows into the `models` table.
  - Updated `public.models` table with specific values based on conditions.

### CODE REVIEW

File: apps/hasura/metadata/databases/masterbots/tables/public_models.yaml
Comment (AndlerRL) [inline apps/hasura/metadata/databases/masterbots/tables/public_models.yaml:1]: @TopETH — Did you create this table a a enum table? I want to make sure before to merge it.

Comment (TopETH) [reply apps/hasura/metadata/databases/masterbots/tables/public_models.yaml:1]: Yes, I did.

Comment (AndlerRL) [reply apps/hasura/metadata/databases/masterbots/tables/public_models.yaml:1]: Ok, awesome! I made a couple comments to the PR #228 related to the type of models. Please take them a look, I saw that there was no model version specification, only the LLMs.

Comment (TopETH) [reply apps/hasura/metadata/databases/masterbots/tables/public_models.yaml:1]: image

Comment (TopETH) [reply apps/hasura/metadata/databases/masterbots/tables/public_models.yaml:1]: image

Comment (TopETH) [reply apps/hasura/metadata/databases/masterbots/tables/public_models.yaml:1]: image

**Review by AndlerRL — APPROVED** [review]
LGTM! Thanks for providing the screenshots and adding the SQL script for the model values. Merging! :+1:

**Comment by TopETH** [comment]
Thanks.

**Pull request merged**